### DISCUSSION TITLE
How to create model and datasource from schema in runtime?

### DISCUSSION THREAD

**Discussion by snowyu** [prisma/prisma#6437] [Q&A] [answered]
Could It add datasource and model dynamically with programming?

**Comment by ryands17** [comment, marked as answer]
@snowyu 👋 

Currently not, but you can create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) here with your use case so that we can look into this.

As a workaround, you can write to `schema.prisma` dynamically via the `fs` module in Node and then run `prisma migrate dev` via `exec` to add the model to the schema.

**Reply by snowyu** [reply]
thanks: #6442.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
